source/_docs/domains.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Domains and Redirects
+description: Understand platform and custom domains and learn how to redirect requests to an alternate Drupal or WordPress domain via PHP.
+tags: [redirects, variables, dns]
+categories: []
+---
+## Platform domains
+Every site on Pantheon is given multiple environments (Dev, Test, and Live), each with their own unique platform domains which are bound to isolated resources, and are uniquely addressable on the internet via Pantheon's own DNS. [Multidev](/docs/multidev/) environments get the same treatment. Platform domains are configured as subdomains, and are generated by prefixing the environment to the site name:
```

Comment:
active voice. Great place to start the doc. What do you think about making this a little less technical?